### ISSUE TITLE
does not generate valid json

### ISSUE THREAD

**Issue by maxandersen** [HanSolo/javafinder#2]
```
javafinder json | jq
parse error: Invalid numeric literal at line 2, column 2
```

the output actually seem to have a bunch of escape codes:

```
{
[0;94m  "timestamp"[0m:[0;95m1685782808[0m,
[0;94m  "search_path"[0m:[0;95m"/System/Volumes/Data/Library/Java/JavaVirtualMachines/"[0m,
[0;94m  "sysinfo"[0m:{
```

**Comment by HanSolo** [comment]
Oh darn of course…was playing around with colored output on the console and forgot to remove it again…tststs

**Comment by HanSolo** [comment]
Fixed with release 17.0.19
